example/App.tsx

```diff
@@ -5,12 +5,15 @@ import { cacheImages, cacheFonts } from './src/helpers/AssetsCaching';
 import vectorFonts from './src/helpers/vector-fonts';
 import { SafeAreaProvider } from 'react-native-safe-area-context';
 import * as SplashScreen from 'expo-splash-screen';
+import { useColorScheme } from 'react-native';
 
 SplashScreen.preventAutoHideAsync();
 
 export default () => {
   const [isReady, setIsReady] = useState(false);
 
+  theme.mode = useColorScheme();
```

Comment:
Don't store the result of `useColorScheme()` directly to theme.mode. Rather do this:

```suggestion
  const colorScheme = useColorScheme();
  theme.mode = colorScheme;
```

This looks better, imo.